dts/bindings/power-domain/power-domain-gpio-monitor.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: Simple GPIO controlled power domain
```

Comment:
Description is wrong, it is not controlled by a GPIO

Reply:
Done